package.json

```
@@ -4,7 +4,10 @@
   "description": "Open Sesame is a platform intended to make contributing to open source less intimidating by matching first-time contributors to experienced mentors.",
   "main": "index.js",
   "scripts": {
-    "jsx-preprocessor": "npx babel --watch src/main/webapp/js/projects/react/ --out-dir src/main/webapp/js/projects/react/jsx-processed --presets react-app/prod"
+    "dev-server": "npm run jsx-preprocessor && mvn package appengine:run || true",
```

Just a Q: What is the benefit of including the || true?

---

Good eye! It's actually pretty interesting. `||` essentially means that if the command to the left throws an error (starting the dev server) then run the command to the right, which is in this case `true` and pretty much just evaluates to nothing. This is used because otherwise every time you did Ctrl+C to close the dev server npm would throw an error.

---

Not very obvious so it's worth commenting. JSON doesn't support comments, maybe we can throw a little Bash comment in the string?
```suggestion
    "dev-server": "npm run jsx-preprocessor && mvn package appengine:run || true  # Don't throw an error after exiting",
```